backend/tests_configs.ini

```diff
@@ -197,6 +197,7 @@ website.base_url = http://localhost:6543
 color.config_file_path = %(here)s/color-test.json
 user.reset_password.validity = 5
 frontend.serve = False
+email.notification.enabled_on_invitation = False
```

Comment:
@inkhey 
In the issue, it is requested to set notifications to true by default. In your MR you add a parameter which is False by default. It is the opposite which is asked.

Reply:
I understand that this is a test file, but, what is the interest to put this here ? By the way, you did not update the reference configuration file (development.ini.sample), so adding this parameter is not visible by user, so he will never configure it, so it is useless to implement a parameterizable behavior.

If you put the parameter in one place only, this is in the development.ini.sample, not in test stuff.